lib/SubscriptionScheduleRevision.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Stripe;
+
+/**
+ * Class Subscription
+ *
+ * @property string $id
+ * @property string $object
+ * @property int $created
+ * @property mixed $invoice_settings
+ * @property boolean $livemode
+ * @property mixed $phases
+ * @property string $previous_revision
+ * @property string $renewal_behavior
+ * @property mixed $renewal_interval
+ * @property string $chedule
+ *
+ * @package Stripe
+ */
+class SubscriptionScheduleRevision extends ApiResource
+{
+
+    const OBJECT_NAME = "subscription_schedule_revision";
+
+    use ApiOperations\All;
+    use ApiOperations\Retrieve;
+
+    /**
+     * @return string The API URL for this Subscription Schedule Revision.
+     */
+    public function instanceUrl()
+    {
+        $id = $this['id'];
+        $schedule = $this['schedule'];
```

Comment:
Soooo I just realized we do a lot of magic in the library to find the parent id. And I worry a liiiiittle bit that this is not documented anywhere (internally at Stripe). And that one day we would rename this to `subscription_schedule` because it seems more consistent and then break half the libraries out there.

I wonder if settling on `parent` for the name across all parent resources would be safer. Just food for thoughts